jenkins-install/docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.7'
+services:
+
+  jenkins:
+    build: .
+    privileged: true
+    user: root
+    ports:
+      - 8080:8080
+      - 50000:50000
+    restart: always
+    container_name: jenkins
+    environment:
+      JAVA_OPTS: -Djenkins.install.runSetupWizard=false
+      #JENKINS_OPTS: --argumentsRealm.roles.user=admin --argumentsRealm.passwd.admin=admin --argumentsRealm.roles.admin=admin
+      JENKINS_ADMIN_ID: admin
+      JENKINS_ADMIN_PASSWORD: admin
```

Comment:
Please change default password and send it to telegram chat. Because it has public IP and accessible on internet. And may be attacked or used by someone, etc.